docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.        out : np.ndarray
</pre><p>Are you okay with the description of <code>out</code>?   </p><p> </p><p>How about this:</p><p>"A 2D numpy array where each row represents a (start, stop) index pair. <code>out[0]</code> corresponds to the chunk immediately preceding index <code>i</code>, and <code>out[-1]</code> corresponds to the left-most chunk in the time series, starting at index 0."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Please ignore it for now as I may revise the function.